models/core/revenuecat_subscription_transactions.sql

```diff
@@ -2,23 +2,37 @@
   config(
     materialized = 'incremental',
     unique_key = 'transaction_row_id',
+    merge_update_columns = ['valid_to'],
     )
 }}
 
-with
+with 
+{% if is_incremental() %} 
+merge_identifiers as (
+  select distinct store_transaction_id 
+        from {{ source('revenuecat', 'transactions') }}
+        where regexp_substr(_file_name, '[0-9]{10}')::timestamp_ntz > (
+            select max(_exported_at) 
+            from {{ this }}
+        )
+),
+{% endif %}
 
 source as (
     select 
         * 
     from 
-        {{ source('revenuecat', 'transactions') }}
+        {{ source('revenuecat', 'transactions') }} src
```

Comment:
why have you aliased this?

Reply:
Good shout I was testing `in` vs `exists` for the incremental logic. For `exists` I needed the alias but I went with `in` so I removed it now :)